src/site/content/en/blog/fugu-status/index.md

```diff
@@ -2,17 +2,12 @@
 title: New capabilities status
 subhead: Web apps should be able to do anything iOS/Android/desktop apps can. The members of the cross-company capabilities project want to make it possible for you to build and deliver apps on the open web that have never been possible before.
 date: 2018-11-12
-updated: 2021-06-08
+updated: 2021-11-03
 tags:
   - blog
   - capabilities
 ---
 
-{% Aside %}
-This is a living document and will be updated as features move between
-states.
-{% endAside %}
-
 The [capabilities project][capabilities-project] is a cross-company effort with the objective of
```

Comment:
This section is almost identical to the subhead, maybe we should go into a bit more detail? Or list some of the apps that have used these new APIs and why? Something along the lines of This work allowed Adobe to bring PS to the web, Excalidraw to deprecate their Electron app, and some other really cool example.